### PR TITLE
🌐 Update Korean translation for `docs/ko/docs/deployment/versions.md`

### PR DESCRIPTION
**Pull request by iCodePoet**
As a Korean, I saw some awkward sentences in the document, so I modified it.
If I have time, I will also proceed with additional translation work.

Thank you for creating a great framework!

### CODE REVIEW

File: docs/ko/docs/index.md [hunk 1]
```diff
@@ -233,31 +233,31 @@ INFO:     Application startup complete.
...
-* _경로_ `/items/{item_id}`는 _경로 매개변수_ `int`형 이어야 하는 `item_id`를 가지고 있습니다.
-* _경로_ `/items/{item_id}`는 선택적인 `str`형 이어야 하는 _경로 매개변수_ `q`를 가지고 있습니다.
+* 두 _경로_ 모두 `GET` (HTTP _methods_ 로 알려진)을 받습니다.
+* _경로_ `/items/{item_id}`는 *경로 매개변수*가 `int`형 이어야 하는 `item_id`를 받습니다.
+* _경로_ `/items/{item_id}`는 optional 로 `str`형 이어야 하는 _경로 매개변수_ `q`를 받습니다.
 
 ### 대화형 API 문서
 
-이제 <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>로 가보십시오.
+이제 <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>로 갑니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:242]: nothing needs to be changed

```suggestion
이제 <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>로 가보십시오.
```

File: docs/ko/docs/index.md [hunk 2]
```diff
@@ -233,31 +233,31 @@ INFO:     Application startup complete.
...
 
-자동 대화형 API 문서를 볼 수 있습니다 (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a> 제공):
+자동 대화형 API 문서를 볼 수 있습니다. (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a> 제공):
 
 ![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
 
 ### 대안 API 문서
 
-그리고 이제 <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>로 가봅시다.
+그리고 이제 <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>로 갑니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:250]: nothing needs to be changed

```suggestion
그리고 이제 <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>로 가봅시다.
```

File: docs/ko/docs/index.md [hunk 3]
```diff
@@ -233,31 +233,31 @@ INFO:     Application startup complete.
...
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
 
 ## 예제 심화
 
-이제 `PUT` 요청에 있는 본문(Body)을 받기 위해 `main.py`를 수정해봅시다.
+이제 `PUT` 요청에 있는 본문(Body)을 받기 위해 `main.py`를 수정 해봅시다.
 
-Pydantic을 이용해 파이썬 표준 타입으로 본문을 선언합니다.
+Pydantic을 이용해 파이썬 표준 타입으로 본문을 선언 합니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:260]:
```suggestion
Pydantic을 이용해 파이썬 표준 타입으로 본문을 선언합니다.
```

File: docs/ko/docs/index.md [hunk 4]
```diff
@@ -233,31 +233,31 @@ INFO:     Application startup complete.
...
 
-다른 자동 문서를 볼 수 있습니다(<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a> 제공):
+다른 자동 문서를 볼 수 있습니다. (<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a> 제공):
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
 
 ## 예제 심화
 
-이제 `PUT` 요청에 있는 본문(Body)을 받기 위해 `main.py`를 수정해봅시다.
+이제 `PUT` 요청에 있는 본문(Body)을 받기 위해 `main.py`를 수정 해봅시다.
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:258]:
```suggestion
이제 `PUT` 요청에 있는 본문(Body)을 받기 위해 `main.py`를 수정해봅시다.
```

File: docs/ko/docs/index.md [hunk 5]
```diff
@@ -365,14 +365,14 @@ item: Item
 
 ---
 
-이전 코드 예제로 돌아가서, **FastAPI**는 다음처럼 처리합니다:
+이전 코드 예제로 돌아가서, **FastAPI**는 다음을 처리합니다:
 
-* `GET` 및 `PUT` 요청에 `item_id`가 경로에 있는지 검증.
-* `GET` 및 `PUT` 요청에 `item_id`가 `int` 타입인지 검증.
+* `GET` 및 `PUT` 요청에 `item_id`가 경로에 있는지를 검증함.
+* `GET` 및 `PUT` 요청에 `item_id`가 `int` 타입인지 검증함
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:371]: missing column.

```suggestion
* `GET` 및 `PUT` 요청에 `item_id`가 `int` 타입인지 검증함.
```

File: docs/ko/docs/index.md [hunk 6]
```diff
@@ -388,7 +388,7 @@ item: Item
 
 우리는 그저 수박 겉핡기만 했을 뿐인데 여러분은 벌써 어떻게 작동하는지 알고 있습니다.
 
-다음 줄을 바꿔보십시오:
+다음 줄을 바꿔봅니다:
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:391]: nothing needs to be changed

```suggestion
다음 줄을 바꿔보십시오:
```

File: docs/ko/docs/index.md [hunk 7]
```diff
@@ -406,13 +406,13 @@ item: Item
         ... "item_price": item.price ...
 ```
 
-...그러고 나서 여러분의 편집기가 속성과 타입을 알고 자동 완성하는지 보십시오:
+...그러고 나서 여러분의 편집기가 속성과 타입을 알고 자동 완성하는지 봅니다:
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:409]: nothing needs to be changed

```suggestion
...그러고 나서 여러분의 편집기가 속성과 타입을 알고 자동 완성하는지 보십시오:
```

File: docs/ko/docs/index.md [hunk 8]
```diff
@@ -406,13 +406,13 @@ item: Item
...
 
 ![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
 
-더 많은 기능을 포함한 보다 완전한 예제의 경우, <a href="https://fastapi.tiangolo.com/tutorial/">튜토리얼 - 사용자 가이드</a>를 보십시오.
+더 많은 기능을 포함한 보다 상세 예제를 확인하고 싶은 경우, <a href="https://fastapi.tiangolo.com/tutorial/">튜토리얼 - 사용자 가이드</a>를 봅니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:413]:
```suggestion
더 많은 기능을 포함한 더 자세한 예제를 확인하고 싶은 경우, <a href="https://fastapi.tiangolo.com/tutorial/">튜토리얼 - 사용자 가이드</a>를 보십시오.
```

File: docs/ko/docs/index.md [hunk 9]
```diff
@@ -429,9 +429,9 @@ item: Item
 
 ## 성능
 
-독립된 TechEmpower 벤치마크에서 Uvicorn에서 작동하는 FastAPI 어플리케이션이 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">사용 가능한 가장 빠른 프레임워크 중 하나</a>로 Starlette와 Uvicorn(FastAPI에서 내부적으로 사용)에만 밑돌고 있습니다. (*)
+독립된 TechEmpower 벤치마크에서 Uvicorn에서 작동하는 FastAPI 어플리케이션이 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">사용 가능한 가장 빠른 프레임워크 중 하나</a>로 Starlette와 Uvicorn(FastAPI에서 내부적으로 사용)에 밑돌고 있습니다. (*)
 
-자세한 내용은 <a href="https://fastapi.tiangolo.com/benchmarks/" class="internal-link" target="_blank">벤치마크</a> 섹션을 보십시오.
+자세한 내용은 <a href="https://fastapi.tiangolo.com/benchmarks/" class="internal-link" target="_blank">벤치마크</a> 섹션을 봅니다.
```
Comment (joonas-yoon) [inline docs/ko/docs/index.md:434]: nothing needs to be changed

```suggestion
자세한 내용은 <a href="https://fastapi.tiangolo.com/benchmarks/" class="internal-link" target="_blank">벤치마크</a> 섹션을 보십시오.
```

**Review by joonas-yoon — COMMENTED** [review]
Thanks for your great works, and I added some comments.

Most of all is about spell check and the way of words (`-합니다` to `-하십시오`)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 229b19989038129dd10b2f5b461ec43a2a6e84ea at: https://649a1239cc19e50f76caccf3--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**